NMS-14349: Add terms to glossary.adoc (#4882)
OpenNMS/opennms: docs/modules/reference/pages/glossary.adoc

{page-component-title} notifications inform users about events in their monitored networks without forcing them to log in and look at the UI.
(See xref:operation:notifications/introduction.adoc[Notifications].)

Observability:: The ability to measure the internal state of a system by examining its output.
If the system's state can be estimated using only its output, it is considered "observable".

Operator board:: Customizable dashboard to visualize monitoring information.
(See xref:operation:admin/webui/opsboard/introduction.adoc[Operator Board].)

> Passive discovery:: The process by which a ``newSuspect`` event is detected by the {page-component-title} Provisiond service and converted into a new node.

Reviewer: Personal note here, but it's strongly recommended NOT to use this, as it just adds the nodes to the database and doesn't give an option to get the nodes into a requisition, which makes it harder to manage them in the long run.

{page-component-title} notifications inform users about events in their monitored networks without forcing them to log in and look at the UI.
(See xref:operation:notifications/introduction.adoc[Notifications].)

Observability:: The ability to measure the internal state of a system by examining its output.

> If the system's state can be estimated using only its output, it is considered "observable".

Reviewer 2: This is a nice addition!

> If the system's state can be estimated using only its output, it is considered "observable".

Reviewer 2: ```suggestion
If the system's state can be estimated using only its output, it is considered "observable".
```
This is a nice addition!

Snaps:: Self-contained software packages that run in a sandbox and have mediated access to host systems.
Our Appliance Service uses snaps to distribute operating system packages and Minion service updates.

> Telemetry:: The process of automatically recording and transmitting data from an external source to a system in a different location for monitoring or analysis.

Reviewer: ```suggestion
Telemetry:: The process of automatically recording and transmitting data from an external source to a system in a different location for monitoring or analysis.
The Telemetry daemon in {page-component-title} accepts data sent from nodes in one of the supported protocol formats.
(See xref:operation:telemetryd/introduction.adoc[Telemetry Daemon].)
```

Time series, time-series database (TSDB):: Time series is a sequence of data points that occur in successive order over a period of time.
A time series database (TSDB) is designed to store and serve time series data.
(See xref:deployment:time-series-storage/timeseries/time-series-storage.adoc[Time Series Storage] and https://en.wikipedia.org/wiki/Time_series_database[Time Series Database].)

Topology:: A description of a network's elements (devices, services) and the connections between them.
(See xref:operation:topology/introduction.adoc[Introduction to Topology].)

> Traps:: Event triggers collected from SNMP-capable devices on the network by the {page-component-title} Trapd service daemon.

Reviewer: ```suggestion
Traps:: Event triggers generated by SNMP-capable devices on the network and send to the {page-component-title} Trapd service daemon.
```

Author: Just fixing a minor tense error in the suggestion. :) 

```suggestion
Traps:: Event triggers generated by SNMP-capable devices on the network and sent to the {page-component-title} Trapd service daemon.
```
